pytube/request.py

```diff
@@ -43,6 +48,50 @@ def get(url, extra_headers=None) -> str:
     return _execute_request(url, headers=extra_headers).read().decode("utf-8")
 
 
+def seq_stream(url, chunk_size=4096, range_size=9437184):
```

Comment:
we should use chunk size default from library-level constant if possible

Reply:
fine by me, I can move some of these defaults to a constants file somewhere. I basically just stole the function declaration from `stream()`, hence those values

Reply:
Can just move to top of this file or class level constant probably since this is only major networking component

Reply:
Moved to top of file in most recent commits